src/lib/getIgnoredUpgradesDueToEnginesNode.ts

```diff
@@ -0,0 +1,49 @@
+import { minVersion, satisfies } from 'semver'
+import { IgnoredUpgradeDueToEnginesNode } from '../types/IgnoredUpgradeDueToEnginesNode'
+import { Index } from '../types/IndexType'
+import { Maybe } from '../types/Maybe'
+import { Options } from '../types/Options'
+import { Version } from '../types/Version'
+import { VersionSpec } from '../types/VersionSpec'
+import getEnginesNodeFromRegistry from './getEnginesNodeFromRegistry'
+import upgradePackageDefinitions from './upgradePackageDefinitions'
+
+/** Checks if package.json min node version satisfies given package engine.node spec */
+const satisfiesNodeEngine = (enginesNode: Maybe<Version>, optionsEnginesNodeMinVersion: string) =>
```

Comment:
You could also reuse satisfiesNodeEngine from [filters.ts](https://github.com/raineorshine/npm-check-updates/blob/f34082d1e84afd54be0a311397dd2473eb2cd8c1/src/package-managers/filters.ts#L28-L41) if you wanted.

Reply:
I could but I would rather not call minVersion a bunch of times and instead call it once and reuse it